### PR TITLE
traceloop loglevel option

### PR DESCRIPTION
**Pull request by alban**
Example:
```
kubectl gadget deploy --traceloop-loglevel=trace,json
```

----

TODO:
- [x] Merge https://github.com/kinvolk/traceloop/pull/44 first
- Tested with traceloop PR 45

### CODE REVIEW

File: cmd/kubectl-gadget/deploy.go [hunk 1]
```diff
@@ -35,6 +36,11 @@ func init() {
 		"traceloop", "",
 		true,
 		"enable the traceloop gadget")
+	deployCmd.PersistentFlags().StringVarP(
+		&traceloopLoglevel,
+		"traceloop-loglevel", "",
```
Comment (mauriciovasquezbernal) [inline cmd/kubectl-gadget/deploy.go:41]: What about having `info` by default to avoid the check in `entrypoint.sh`?

Comment (alban) [reply cmd/kubectl-gadget/deploy.go:41]: The default is "info,json" in the line below. The `""` here is about the lack short version of the option (no `-t`).

Comment (alban) [reply cmd/kubectl-gadget/deploy.go:41]: Do you want to change the default or to remove the check in `entrypoint.sh`?

Comment (mauriciovasquezbernal) [reply cmd/kubectl-gadget/deploy.go:41]: oh right, I read the wrong field. I think it's fine to keep the check that as well.

**Comment by alban** [comment]
Thanks! I updated the reference to the last commit in traceloop and rebased. I'll merge when the CI is green.

**Pull request merged**